### PR TITLE
Material database patch

### PR DESCRIPTION
**Pull request by Lawproto**
As of https://github.com/icub-tech-iit/cad-mechanics/issues/244, this is the new list of materials.

### CODE REVIEW

**Comment by Lawproto** [comment]
This is the full list, for easier reading:
```
3d-systems_duraform_prox_pa.mtl
3d-systems_laserform_316l.mtl
aa7075-t6.mtl
aisi_302.mtl
aisi_304.mtl
aisi_316.mtl
aisi_316l.mtl
alm_pa_603-cf.mtl
apworks_alsi10mg.mtl
apworks_scalmalloy.mtl
apworks_ti6al4v.mtl
dupont_delrin_100_nc010.mtl
dupont_rynite_555_nc010a.mtl
eltr_comp_mat.mtl
ge_lexan_9030.mtl
igus_iglidur_j.mtl
igus_iglidur_w300.mtl
iso1043_pe_500.mtl
iso1629_nbr_70-shore-a.mtl
iso1629_sbr_70-shore-a.mtl
iso3506_a2-70.mtl
rohm_plexiglas_8n.mtl
stratasys_rgd720.mtl
stratasys_tangoblackplus_flx980.mtl
stratasys_vero.mtl
tecacomp_peek_lds.mtl
ul94v-0_fr-4.mtl
uns_c38500.mtl
uns_c51000.mtl

```

**Comment by Lawproto** [comment]
@fiorisi I ping you to review this.

**Comment by fiorisi** [comment]
@Lawproto very nice!
Did you add also the AISI 630? Here you can find the datasheet: https://github.com/icub-tech-iit/cad-mechanics/wiki/Metal-materials

Concerning the bronze there was the [ISO 1338:1977](https://www.iso.org/standard/5904.html) which status is `Withdrawn`.
So I would stick to the UNS designation.

**Comment by fiorisi** [comment]
You can also have a look here: https://www.casadelbronzo.com/en/bronze-alloys-mechanical-properties/copper-alloys-designation/

**Comment by Lawproto** [comment]
> Did you add also the AISI 630?

@fiorisi done.

File: config/materials/aisi_630.mtl [hunk 1]
```diff
@@ -0,0 +1,86 @@
+﻿ND_RelParSet_K01 = {
+
+Name = AISI_630
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'Stainless steel with excellent mech props, cond. H 1075. Aka 17-4PH.
```
Comment (fiorisi) [inline config/materials/aisi_630.mtl:9]: @Lawproto usually we purchase the material in `Condition A`. See as a reference this issue https://github.com/icub-tech-iit/cad-mechanics/issues/34 and the associated brochure. 
I would keep the post-processing heat treatments for the dedicated section in the drawing frame.

File: config/materials/aisi_630.mtl [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+{
+  Name = PTC_YOUNG_MODULUS
+  Type = Real
+  Default = 1.970000e+02 GPa
+  Access = Full
+},
+{
+  Name = PTC_TENSILE_YIELD_STRESS
+  Type = Real
+  Default = 1.020000e+03 MPa
```
Comment (fiorisi) [inline config/materials/aisi_630.mtl:76]: Can you check this numbers coherently with the previous comment?

**Review by fiorisi — CHANGES_REQUESTED** [review]
See the following comments.

File: config/materials/aisi_630.mtl (hunk 1 above)
Comment (Lawproto) [reply config/materials/aisi_630.mtl]: Ok, I am assuming the A stands for Annealed.

File: config/materials/aisi_630.mtl (hunk 2 above)
Comment (Lawproto) [reply config/materials/aisi_630.mtl]: Yes, sure.

**Comment by Lawproto** [comment]
@fiorisi I pushed changes in a new commit without using the interactive interface of the PR, my bad!
Can you please take a look and check your instructions are implemented, anyway?

File: config/materials/aisi_630.mtl [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+{
+  Name = PTC_MASS_DENSITY
+  Type = Real
+  Default = 7.750000e+00 g/cm^3
+  Access = Full
+},
+{
+  Name = PTC_POISSON_RATIO
+  Type = Real
+  Default = 2.720000e-01
```
Comment (fiorisi) [inline config/materials/aisi_630.mtl:64]: @Lawproto as discussed F2F I would use the numbers in the material datasheet.

**Review by fiorisi — CHANGES_REQUESTED** [review]

File: config/materials/aisi_630.mtl (hunk 3 above)
Comment (Lawproto) [reply config/materials/aisi_630.mtl]: @fiorisi ok, changes implemented in the new commit. According to the datasheet, cond. A falls between H 1100 and H 1150 in terms of mechanical properties so I did an average between the two, when the cond. A value was not available directly.

immagine

**Comment by Lawproto** [comment]
Ok, we are ready to implement this new material database. Let us just wait the next @icub-tech-iit/silo-mech meeting to explain the situation then merge.

**Pull request merged**